src/client/App.jsx

```diff
@@ -8,22 +8,92 @@ const apiUrl = 'http://localhost:4000';
 function App() {
   const [movies, setMovies] = useState([]);
 
+
   useEffect(() => {
-    fetch(`${apiUrl}/movie`)
+    const options = {
```

Comment:
Interesting use of adding auth to the useEffect, was this to only load the movies for a particular user once logged in?

Reply:
Yeah that was the idea.